compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRNonTerminator.java

```diff
@@ -192,20 +192,23 @@ public BIROperand[] getRhsOperands() {
     public static class NewStructure extends BIRNonTerminator {
         public BIROperand rhsOp;
         public List<BIRMappingConstructorEntry> initialValues;
+        public BType type;
```

Comment:
@warunalakshitha, is it OK to have AST/FE types in the BIR? There are some other places where this is already done, but just wondering if it is ideal.

Reply:
We already have BType in Bir java model and also type attributes are added to bir binary. IMO it should be ok